modules/extensions-spring-support/src/test/java/org/mule/module/extension/internal/connector/PetStoreConnectionValidationTestCase.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.connector;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+
+import org.mule.api.MessagingException;
+
+import org.junit.Test;
+
+public class PetStoreConnectionValidationTestCase extends PetStoreConnectionTestCase
+{
+
+    private static final String INVALID_CREDENTIALS_ERROR_MESSAGE = "Invalid credentials.";
+
+    @Override
+    protected String getConfigFile()
+    {
+        return "petstore-simple-connection.xml";
+    }
+
+    @Test
+    public void getInvalidConnection() throws Exception
+    {
+        try
+        {
+            runFlow("getPetsWithInvalidConfigWithConnectionValidation").getMessage().getPayload();
```

Comment:
if this doesn't fail the test still passes. Rewrite this tests using the ExpectedException rule from JUnit